beacon/goclient/duties.go

```diff
@@ -38,7 +40,7 @@ func (gc *goClient) GetDuties(epoch phase0.Epoch, validatorIndices []phase0.Vali
 	}
 	wg.Wait()
 
-	gc.logger.Debug("fetched duties", zap.Int("count", len(duties)), zap.Float64("duration (sec)", time.Since(start).Seconds()))
+	gc.logger.Debug("fetched duties", zap.Int("count", len(duties)), logging.DurationNano(start))
```

Comment:
should be DurationSecond

Reply:
is using seconds prefered for any reason?

Reply:
I think nano is too small scope. MiliSec is better IMO